docs/src/corpus.md

```diff
@@ -83,3 +139,56 @@ corpus. The easiest way to do this is to convert a `Corpus` object into
 a `DataFrame`:
 
     convert(DataFrame, crps)
+
+## Corpus Metadata
+
+You can also retrieve the metadata for every document in a `Corpus` at once:
+
+* `languages()`: What language is the document in? Defaults to `Languages.English()`, a Language instance defined by the Languages package.
+* `names()`: What is the name of the document? Defaults to `"Unnamed Document"`.
```

Comment:
I think `names` should be changed to `titles`.  I was earlier unclear what `names` refers to.

Reply:
Do you prefer that I send another PR making changes, `name` -> `title` & `name!` -> `title!` & `names` -> `titles` & `names!` to `titles!` (across the entire codebase, including tests, docs) ? Or include in this one itself?

Reply:
I am marking this resolved as per #135